deftree/build_deftree.go

```diff
@@ -92,30 +92,30 @@ func New(req *plugin.CodeGeneratorRequest, serviceFile io.Reader) (Deftree, erro
 	return &dt, nil
 }
 
-func NewFromString(def string) (Deftree, error) {
+func NewFromString(def, GOPATH string) (Deftree, error) {
```

Comment:
I don't feel too strongly either way; the mixed examples from the Go code itself satisfy me.